Applications/Custom/LayerClient/jni/pow.h

```diff
@@ -0,0 +1,119 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2020 Jihoon Lee <jhoon.it.lee@samsung.com>
+ *
+ * @file   pow.h
+ * @date   16 November 2020
+ * @brief  This file contains the simple pow2 layer which squares input
+ * elements.
+ * @see    https://github.com/nnstreamer/nntrainer
+ * @author Jihoon Lee <jhoon.it.lee@samsung.com>
+ * @bug    No known bugs except for NYI items
+ *
+ */
+
+#ifndef __POW_LAYER_H__
+#define __POW_LAYER_H__
+
+#include <layer.h>
+#include <tensor.h>
+
+namespace custom {
+class PowLayer : public ml::train::Layer {
+public:
+  /**
+   * @brief Construct a new Pow Layer object that does elementwise power
+   *
+   * @param exponent_ exponent
+   */
+  PowLayer(float exponent_ = 1) : exponent(exponent_) {}
+
+  /**
+   * @brief Destroy the Pow Layer object
+   *
+   */
+  ~PowLayer();
+
+  /**
+   * @brief     set Property of layer
+   * @param[in] values values of property
+   * @retval #ML_ERROR_NONE Successful.
+   * @retval #ML_ERROR_INVALID_PARAMETER invalid parameter.
+   */
+  int setProperty(std::vector<std::string> values) {
+    /**< NYI */
+    return 1;
+  }
+
+  /**
+   * @brief Set the Property by propertyType, this is not used in this demo
+   *
+   * @param type property type
+   * @param value value
+   */
+  void setProperty(const ml::train::Layer::PropertyType type,
+                   const std::string &value = "") {
+    /**< NOT USED */
+  }
+
+  /**
+   * @brief check if hyperparameter is valid
+   *
+   * @return int ML_ERROR_NONE if successful
+   */
+  int checkValidataion() { return ML_ERROR_NONE; }
```

Comment:
`checkValidation`

Reply:
This is fixed by inheriting layer_internal.h and the function is deleted.